DependencyInjection/Configuration.php

```diff
@@ -86,6 +86,7 @@ private function getMailersNode()
             ->children()
                 ->scalarNode('url')->defaultNull()->end()
                 ->scalarNode('transport')->defaultValue('smtp')->end()
+                ->scalarNode('command')->defaultValue('/usr/sbin/sendmail -bs')->end()
```

Comment:
Maybe `sendmail_command` because it is so specific and exists only for that transport?